integration-tests/src/test/scala/tech/beshu/ror/integration/suites/ResponseFieldRuleSuite.scala

```diff
@@ -72,7 +72,7 @@ class ResponseFieldRuleSuite
       val result = searchManager.search(
         ujson.read("""{"query": {"terms":{"user_id": ["alice", "bob"]}}}""")
       )
-      result.responseCode should be(200)
+      result should have statusCode 401
```

Comment:
Status has changed from 200 to 401. Is this OK?

Reply:
thanks for catching this. It was a problem with the custom matcher